doc/examples/color_exposure/plot_adapt_rgb.py

```diff
@@ -35,11 +35,10 @@ def sobel_each(image):
 def sobel_hsv(image):
     return filters.sobel(image)
 
-"""
-We can use these functions as we would normally use them, but now they work
-with both gray-scale and color images. Let's plot the results with a color
-image:
-"""
+######################################################################
+# We can use these functions as we would normally use them, but now they work
+# with both gray-scale and color images. Let's plot the results with a color
+# image:
```

Comment:
@emmanuelle sorry, I haven't been following this PR, but is this format change necessary? I think it's way uglier to read!